virtcontainers/store/filesystem.go

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) 2019 Intel Corporation
```

Comment:
It would be clearer if this was called `virtcontainers/store/filesystem_backend.go`.

Reply:
Maybe `fs_backend.go` for brevity.

Reply:
Renamed to `filesystem_backend.go`, thanks.